File: ClientApp/src/models/Post.ts
export class Post {
  ID: number;
  Title: string;
  Content: string;
  Date: string;
  Author: string;
  AuthorID: number;
  constructor( title: string, content: string, date: string, author: string,  authorID: number) {

    this.Title = title;
    this.Content = content;
    this.Date = date;
    this.Author=author;
    this.AuthorID=authorID;
  }
}

File: Models/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend_Foto.Models
{
    public class Post
    {

        public int ID { get; set; }
        public string Title { get; set; }
        public string Content{ get; set; }
        public string Date { get; set; }
        public string Author { get; set; }
        public int  CommentCount { get; set; }
        public int AuthorID { get; set; }

    }
}
